extensions/datasketches/src/main/java/io/druid/query/aggregation/datasketches/theta/SketchMergeAggregatorFactory.java

```diff
@@ -81,7 +85,8 @@ public AggregatorFactory getMergingFactory(AggregatorFactory other) throws Aggre
           name,
           Math.max(size, castedOther.size),
           shouldFinalize,
-          true
+          true,
+          errorBoundsStdDev
```

Comment:
we should check that errorBoundsStdDev for "other" was same as this.errorBoundsStdDev or else should fail with AggregatorFactoryNotMergeableException .